crates/tree-sitter-qmd/tree-sitter-markdown/src/scanner.c

```diff
@@ -2,7 +2,6 @@
 #include <assert.h>
 #include <ctype.h>
 #include <string.h>
-#include <wchar.h>
```

Comment:
This one didn't seem used, and the shims didn't provide a shim for it, so I removed